김호집/PROGRAMMERS/java/Week11/구명보트.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Author    : HojipKim
+ * Date      : 2024.08.03(Sat)
+ * Runtime   : 35.73 ms
+ * Memory    : 55 MB
+ * Algorithm : 탐욕법（Greedy）
+ */
+class Solution {
+    public int solution(int[] people, int limit) {
+        int answer = 0;
+
+        ArrayList<Integer> arr = new ArrayList<>();
+
+        for(int i : people){
+            arr.add(i);
+        }
```

Comment:
```suggestion
        ArrayList<Integer> arr = Arrays.stream(people).boxed().collect(Collectors.toList());
```

반복문을 내부 반복으로 감추는 것도 좋은 코드 개선 방향일 것 같습니다.

하지만 이 문제에서 가장 최선은 아래와 같은 이유로 주어진 배열을 그대로 이용하는 것이라고 생각 합니다.

- 추가적인 메모리 사용
- 배열에 제공되는 정렬 API Arrays.sort() 를 사용 가능